designs/administrator/future-openshift/cluster-settings/index.md

```diff
@@ -0,0 +1,64 @@
+---
+parent: Adminstrator
+version: 4.6
+---
+
+# Cluster Settings 
+
+Users complete upgrades in cluster settings. The following design focuses on making improvements to the current upgrade experience in cluster settings by making upgrading as simple and transparent as possible. Specific improvements include adding more information around recommended upgrade paths and notifying users when new versions and channels become available. 
+
+## Adding Alerts to the Notification Drawer
+
+![Alerts - Available Updates](img/notifications-updates.png)
+- Today, available updates appear in the message section of the notification drawer. 
+- Three new alerts will be added to inform users if (1) a new patch becomes available, (2) a new minor release becomes available, and/or (3) new channels become available. 
+- All three messages send users to the cluster settings section of the console to take further action.
+
+## Channel Visualization and In Context Help
+
+The new channel visualization provides users with more transparency into recommended upgrade paths, informs users what changes happen in each release, and guides users to the correct channel. 
+
+In addition to the visualization: 
+- Field level help text was added for channels
+- A new manage subscription field was added with a direct link to manage subscriptions in OpenShift Cluster Manager
+
+![Channel help text](img/channel-help.png)
+- Clicking `Channel` surfaces a popover that provides context as to what a channel is along  with a link to documentation to learn more about channels in OpenShift.
+
+![Channel visualization](img/channel-visualization.png)
+- The blue line signifies the current channel you are on. The current version and recommended version (if available) are both marked with blue dots. 
```

Comment:
a few questions comments about this:
1. That grey doesn't look like it passes contrast - did you test for a11y on it?
2. the "+ more" COULD be a PF button right? Did you try using a standard secondary button design here rather than introduce a new style? (It might not look right but curious :) )
3. Did you consider adding a simple legend here to explain the colors? I really like how clear you document this and I wonder if it would be helpful to a user to see that same explanation.

Reply:
@matthewcarleton 
1. Yes it does! I used an existing border color from PF - check [this visual spec I did here](https://marvelapp.com/prototype/64j205c/screen/69870478). Gives allllll the info.
2. We didn't try using a secondary button but I think it would look massive in this situation since the current height of the "more button" is half the height (18px rather than 36)
3. We didn't try adding a legend although it could be a good addition depending on how well it fits in. @megan-hall I know you presented this is some feedback sessions - did users struggle figuring out what each of the lines meant?

Reply:
@gdoyle1 @matthewcarleton so far no feedback on understanding the difference between the lines. I am working on an enhancement story - I will add this to the list and see how others feel about adding a key. Great idea Matt!

Reply:
> @matthewcarleton
> 
> 1. Yes it does! I used an existing border color from PF - check [this visual spec I did here](https://marvelapp.com/prototype/64j205c/screen/69870478). Gives allllll the info.
oh nice work on the spec :) So are you saying it passes b/c it's a border color from PF? I checked #d2d2d2 on a white bg and it fails but I might be missing something. I don't think all PF border colors pass b/c some of them don't need to. 
> 2. We didn't try using a secondary button but I think it would look massive in this situation since the current height of the "more button" is half the height (18px rather than 36)
ah ya that's where it gets tricky b/c the buttons are a height that is required to pass a11y for clickable area. I see your point though about it being much larger 🤔  this comes down to rules around when we require a11y or when we are ok with not meeting the AA requirement. I'm not sure how to determine the best path forward for it. Maybe consult with PF a11y? (sorry to be the accessibility squeaky wheel)
> 3. We didn't try adding a legend although it could be a good addition depending on how well it fits in. @megan-hall I know you presented this is some feedback sessions - did users struggle figuring out what each of the lines meant?
👍

Reply:
@matthewcarleton @gdoyle1 updated the gray line to comply with a11y contrast and increased the "+ more" to match the hight of PF labels (24px).

Reply:
@megan-hall Looks great !!

Reply:
awesome! one small thing :) Should the text for both channels be the same colour? Does the inactive channel text colour pass a11y?

Reply:
We used the same color of the gray line (though I know it looks lighter)

 --pf-global--disabled-color--100 (#6A6E73)